pkgs/applications/networking/protonmail-bridge/default.nix

```diff
@@ -55,7 +55,7 @@ in mkDerivation {
       --set-rpath "${rpath}" \
       $out/lib/protonmail-bridge
 
-    substituteInPlace $out/share/applications/ProtonMail_Bridge.desktop \
+    substituteInPlace $out/share/applications/protonmail_bridge.desktop \
```

Comment:
`s/protonmail_bridge/protonmail-bridge/`

Reply:
Oh! Thank you.